lib/embed.dart

```diff
@@ -646,7 +646,14 @@ class Embed extends EditorUi {
   }
 
   void _handleOpenInDartPad() {
-    window.open('/embed-$_modeName.html?id=$gistId', 'DartPad_$gistId');
+    final Map<String, String> currentParams = queryParams.parameters;
+    String paramStr = '';
+    currentParams.forEach((key, value) {
+      // get rest of original params to pass to new window
+      paramStr += (paramStr == '' ? '' : '&') + '$key=$value';
+    });
+    final String uriStr = Uri.encodeFull('embed-$_modeName.html?$paramStr');
+    window.open(uriStr, 'DartPad_$gistId');
```

Comment:
We should probably change this to use the Uri class to do the query parameter formatting: https://api.flutter.dev/flutter/dart-core/Uri/replace.html

Reply:
As I wrote that code I knew I was being lazy and not looking up a uri method.. lol
But.. on further analysis, I think all we are doing here is re-creating the exact same URL which the iframe was opened with.  The 'embed-$_modeName.html' string is going to equate to the original html page, and we want all the query parameters, so it seems that this method can simply be:

    window.open(window.location.href, 'DartPad_$gistId');